File: LorePdks.COMMON/DTO/Hareket/HareketDTO.cs
using LorePdks.COMMON.DTO.Base;
using LorePdks.COMMON.DTO.Common;
using LorePdks.COMMON.DTO.Firma;
using System;
using System.Collections.Generic;
using System.Linq;
using System.Text;
using System.Threading.Tasks;

namespace LorePdks.COMMON.DTO.Hareket
{


    public class HareketDTO : BaseDTO
    {
        public FirmaDTO firmaDto { get; set; }
        public KodDTO hareketTipKodDto { get; set; }
        public KodDTO hareketDurumKodDto { get; set; }
        public DateTime? hareketKayitTarih { get; set; }
        public DateTime? hareketIslemeTarih { get; set; }
        public string hareketdata { get; set; }
    }
}

File: LorePdks.WEB/src/app/core/models/HareketDto.ts
import { baseDto } from "./baseDto";
import { FirmaDto } from "./FirmaDto";
import { KodDto } from "./KodDto";

/**
 * Hareket bilgisini temsil eden model
 */
export class HareketDto extends baseDto {
  firmaDto: FirmaDto;
  hareketTipKodDto: KodDto;
  hareketDurumKodDto: KodDto;
  hareketKayitTarih?: Date;
  hareketIslemeTarih?: Date;
  hareketdata: string;
}